DESCRIPTION

```diff
@@ -22,6 +22,7 @@ Imports:
     jsonlite,
     openssl,
     rlang,
+    rstudioapi,
```

Comment:
can you bump the version to 1.99.11 so we can see if the users have the version with this feature please